### ISSUE TITLE
Build fails on nodejs 10

### ISSUE THREAD

**Issue by Apollon77** [OpenZWave/node-openzwave-shared#227]
see https://travis-ci.org/ioBroker/ioBroker.zwave/jobs/371390543#L623:

Basically it is error:

```
In file included from /home/travis/build/ioBroker/ioBroker.zwave/node_modules/nan/nan.h:192:0,
                 from ../src/openzwave.hpp:27,
                 from ../src/callbacks.cc:18:
/home/travis/build/ioBroker/ioBroker.zwave/node_modules/nan/nan_maybe_43_inl.h: In function ‘Nan::Maybe<bool> Nan::ForceSet(v8::Local<v8::Object>, v8::Local<v8::Value>, v8::Local<v8::Value>, v8::PropertyAttribute)’:
/home/travis/build/ioBroker/ioBroker.zwave/node_modules/nan/nan_maybe_43_inl.h:112:15: error: ‘class v8::Object’ has no member named ‘ForceSet’
   return obj->ForceSet(isolate->GetCurrentContext(), key, value, attribs);
               ^
```

... and some more warnings

**Comment by codepeon** [comment]
try https://github.com/OpenZWave/node-openzwave-shared/pull/229 it should fix the build error

**Comment by Apollon77** [comment]
see comment on PR: worked https://travis-ci.org/Apollon77/node-openzwave-shared/builds/373208967

**Comment by ekarak** [comment]
version 1.4.2 released with NAN 2.10.0, resolving issue
